classes/article/PublishedArticleDAO.inc.php

```diff
@@ -221,6 +303,10 @@ function getPublishedArticlesInSections($issueId, $useCache = false) {
 					$publishedArticles[$currSectionId]['title'] = $publishedArticle->getSectionTitle();
 				}
 			}
+
+			// get galleys of the current version
+			$publishedArticle->setGalleys($this->galleyDao->getBySubmissionId($row['submission_id'], null, $publishedArticle->getCurrentVersionId())->toArray());
```

Comment:
do we not get this already above with _fromRow? -- if we look after the MAX(ss.submission_revision)?